packages/backend-core/tests/core/utilities/testContainerUtils.ts

```diff
@@ -1,85 +1,66 @@
+import { DatabaseImpl } from "../../../src/db"
```

Comment:
I've changed how this file works because when I moved away from `jest-testcontainers`, this code started crafting invalid CouchDB URLs.

The new code uses `docker ps --format json` to make parsing the output easier, and it also makes sure to always return `testcontainers` containers by looking for the `org.testcontainers=true` label.

Reply:
This is a nice improvement, I didn't even know you could get JSON out here, very handy!